lightkurve/tests/test_periodogram.py

```diff
@@ -227,7 +231,8 @@ def test_bls(caplog):
     keys = ['period', 'power', 'duration', 'transit_time', 'depth', 'snr']
     assert np.all([key in  dir(p) for key in keys])
 
-    p.plot()
+    ax = p.plot()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'ax' is assigned to but never used](https://app.codacy.com/manual/barentsen/lightkurve/pullRequest?prid=4938792)